### DISCUSSION TITLE
When the parent element contains the 【overflow: auto;】 property, I got a bug!

### DISCUSSION THREAD

**Discussion by zym-mcarlo** [Akryum/v-tooltip#671] [Q&A]
When setting the tooltip element, its parent element has overflow: auto; Property, and the height of the parent element is less than 50px，The tooltip box is overlaying the element.

**Comment by MiniGab** [comment]
I had the same issue and found the solution on the popper.js github : https://github.com/popperjs/popper-core/issues/248

Here it's what my code looks like :
```
<div 
  v-tooltip.right-start="{
    content: 'hello world',
    popperOptions: {
      modifiers: {
        preventOverflow: {
          boundariesElement: 'offsetParent',
        },
      },
    },
  }"
/>
```

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/4539476/79767112-2bcd6400-8329-11ea-9059-7e6e81d14c1f.png)

@MiniGab nice one, but it seems to impact the responsiveness

**Comment by ghost** [comment]
I guess you have to choose between the two :)
